lib/rubygems/commands/audit_command.rb

```diff
@@ -0,0 +1,93 @@
+# frozen_string_literal: true
+require 'rubygems/command'
+require 'rubygems/package'
+require 'rubygems/local_remote_options'
+require 'rubygems/version_option'
+require 'rubygems/gemcutter_utilities'
+require 'rubygems/advisory_option'
+
+class Gem::Commands::AuditCommand < Gem::Command
+  include Gem::VersionOption
+  include Gem::GemcutterUtilities
+  include Gem::AdvisoryOption
+
+  def initialize
+    super 'audit', 'check if advisories are present'
+
+    add_version_option
+    add_platform_option
+    add_ignore_cve_option
+  end
+
+  def arguments # :nodoc:
+    "GEM  name of the gem to be checked for vulnerabilities"
+  end
+
+  def description # :nodoc:
+    <<-EOF
+The audit command allows you to check advisories reported for a gem version.
+
+An advisory can be any vulnerability in the gem that makes the gem
+either unusable or risky to use. The risk can or cannot be a security issue.
+
+    EOF
+  end
+
+  def usage # :nodoc:
+    "#{program_name} GEM -v VERSION [-p PLATFORM]"
```

Comment:
should this allow being called without a gem (or a version) to audit the set of all installed gems?

Reply:
@segiddins Would that duplicate the function of `bundle audit` except being for all gems on system and not just for bundled ones?